examples/flagsmith-on-ecs/terraform/11_rds.tf

```diff
@@ -0,0 +1,27 @@
+
+resource "aws_db_subnet_group" "production" {
+  name = "main"
+  # subnet_ids = module.vpc.public_subnets
```

Comment:
I think this line should probably go away, as I think we'd always want to encourage using private subnets for a database.